torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -7,7 +7,7 @@ namespace torch { namespace utils {
 PyObject* tensor_to_numpy(const at::Tensor& tensor) {
   throw std::runtime_error("PyTorch was compiled without NumPy support");
 }
-at::Tensor tensor_from_numpy(PyObject* obj) {
+at::Tensor tensor_from_numpy(PyObject* obj, bool copy_numpy=false) {
```

Comment:
should we also remove defaults here?

Reply:
But if the function is called without passing this argument it may fail. And by default, we can assume that a copy is not created and raise a warning for that.